docs/api/covidcast-signals/quidel.md

```diff
@@ -1,12 +1,138 @@
 ---
 title: Quidel
-parent: Inactive Signals
+parent: Data Sources and Signals
 grand_parent: COVIDcast API
 ---
 
 # Quidel
+{: .no_toc}
 
 * **Source name:** `quidel`
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## COVID-19 tests
+
+* **First issued:** 
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+
+Data source based on COVID-19 Antigen tests, provided to us by Quidel, Inc. When
+a patient (whether at a doctor’s office, clinic, or hospital) has COVID-like
+symptoms, doctors may order an antigen test to confirm the presence of the
+virus. Quidel began providing us with test data starting May 9, 2020, and data
+volume increased to statistically meaningful levels starting May 26, 2020.
```

Comment:
It'd be helpful to say explicitly in this paragraph that antigen tests detect current infections, not prior ones -- or whatever description is true.

Reply:
any thoughts on what would work better than "presence of the virus"?

Reply:
"to confirm the patient is currently infected with the virus"? That at least has "currently", to contrast against antibody tests